vizro-core/src/vizro/managers/_data_manager.py

```diff
@@ -28,14 +48,14 @@ class DataManager:
     _cache = Cache(config={"CACHE_TYPE": "SimpleCache"})
```

Comment:
Now that we are using `self` in the memoize decorator, we should be able to move this into the `__init__` method so it's an instance rather than class property, which is really where it should always have gone.